### ISSUE TITLE
Support for APCA values (WCAG 3.0)

### ISSUE THREAD

**Issue by AlanGornick** [ThePacielloGroup/CCAe#232]
I'm hoping you'll include a WCAG 3.0 result for the new APCA measurement system. This would be a huge help moving forward into 3.0!

https://twitter.com/danhollick/status/1468958644364402702?s=21

**Comment by ferllings** [comment]
Yes, that's my plan to include it, as an option until it gets fully approved.

**Comment by Myndex** [comment]
Hi @AlanGornick and @ferllings 

I just want to mention that we released Bridge-PCA recently. It is backwards compatible with WCAG 2 contrast, but is using APCA technology.

### Both Bridge-PCA and the W3 version of APCA are available as NPM packages

    npm i apca-w3

    npm i bridge-pca

The GitHub repo for all issues and discussion of APCA is: [Myndex/SAPC-APCA/discussions](https://github.com/Myndex/SAPC-APCA/discussions)

There is a simple demo of Bridge-PCA, with more information on usage at https://www.myndex.com/BPCA/


Thank you!

Andy
